src/content/structured/components/dialog/code.mdx

```diff
@@ -813,13 +813,9 @@ const termsDialogConfirmed = (ev) => {
 return (
   <>
     <IcButton onClick={handleDialogOpen}>Launch dialog</IcButton>
-    <IcDialog
-      open={openDialog}
-      label="Coffee order"
+    <IcDialog ref={dialogEl} label="Coffee order"
```

Comment:
I'm not quite sure what the change you intended to make here is as there's no issue linked in the PR description. 
Going by the intended change in the commit message: A ref isn't an event listener. A ref also needs to be initiated using a useRef, which is not currently present in this particular code example